client/my-sites/plugins/plans/index.tsx

```diff
@@ -51,6 +71,19 @@ const Plans = () => {
 				subHeaderText={ `Choose the plan that's right for you and reimagine what's possible with plugins` }
 				brandFont
 			/>
+			<div className="plans">
+				<PlansFeaturesMain
+					site={ selectedSite }
+					isInSignup={ false }
+					isInMarketplace
+					intervalType={ intervalType }
+					onUpgradeClick={ onSelectPlan }
+					flowName={ 'marketplace' }
```

Comment:
What's the difference between `flowName={ 'marketplace' }` add `isInMarketplace`? Are they being used differently?